commons/utils/jira_utils.py

```diff
@@ -82,12 +82,16 @@ def get_test_ids_from_te(self, test_exe_id, status=None):
                         te_tag = te_tag.lower()
                     req_success = True
             except (JIRAError, requests.exceptions.RequestException) as fault:
+                if fault.status_code == HTTPStatus.UNAUTHORIZED:
+                    raise EnvironmentError("Unauthorized JIRA credentials") from fault
                 print('Error occurred in getting te tag')
-                LOGGER.error(f'Error occurred {fault} in getting te_tag from {test_exe_id}')
+                LOGGER.error('Error occurred %s in getting te_tag from %s', fault, test_exe_id)
                 retries_cnt = retries_cnt - 1
                 retry_attempt = retry_attempt + 1
                 time.sleep(incremental_timeout_sec * retry_attempt)
-
+                if not retries_cnt:
```

Comment:
was just thinking that we might in future think that which other errors need to be added for exception early like we did for unauthorized but for now this looks good. So even if we do not have line 85/86, even the unauthorized will fail eventulally after15 mins.